ertviz/models/plot_model.py

```diff
@@ -3,6 +3,41 @@
 import plotly.figure_factory as ff
 
 
+class BoxPlotModel:
+    def __init__(self, **kwargs):
+        self.selected = True
+        self._x_axis = kwargs["x_axis"]
+        self._y_axis = kwargs["y_axis"]
+        self._text = kwargs["text"]
+        self._name = kwargs["name"]
+        self._color = kwargs["color"]
+
+    @property
+    def repr(self):
+        repr_dict = dict(
+            # x=self._x_axis,
+            y=self._y_axis,
+            # text=self.display_name,
```

Comment:
remove

Reply:
Removing for now, left another comment for `x=self._x_axis` which we should discuss on our next presentation.